core/src/avm2/globals/XML.as

```diff
@@ -49,6 +49,10 @@ package {
         AS3 native function text():XMLList;
         AS3 native function toString():String;
 
+        public function length(): int {
```

Comment:
This should be an `AS3` function instead of a `public` function, and should be declared on the prototype in the same way that `toXMLString` is.

Reply:
Thank you, and sorry about the wrong implementation before, I know very little about ActionScript. Does the new implementation look correct?